_pages/a-pairing-session-template.md

```diff
@@ -0,0 +1,116 @@
+---
+layout: default
+title: A Pairing Session Template
+permalink: /template
+meta: A template to help guide your pair programming session
+---
+
+{% include article-top.html %}
+
+Here's a template for your next pairing session:
+
+```
+1. [ ] Agree on the high-level goal out loud.
+2. [ ] Break the work into a handful of tasks.
+3. [ ] Come up with a preliminary ordering for those tasks.
+4. [ ] Decide your driver/navigator swapping strategy.
+5. [ ] Configure git to share credit.
+6. [ ] Eliminate distractions.
+7. [ ] Work.
+8. [ ] Analyze the session.
+```
+
+Let's talk more about each step.
+
+
+## 1. Agree on the high-level goal
+
+State **out loud** what you hope to accomplish at a high level.
+
+You wouldn't think it'd be possible for two people to start pairing without agreement about where they're headed, but it's surprisingly easy.
+
+
+## 2. Break the goal into smaller tasks and order them
+
+It's worth trying to break your high-level goal into a handful of smaller steps.
+
+This has a number of benefits:
+
+- It makes the goal less intimidating.
+- You'll spot dead ends and pitfalls more easily.
+- You can sort your task list by priority.
```

Comment:
Also, it provides a nice parking lot.  When you're in the middle of the work flow, something might occur to the pair: "Oh, we should probably do X as well for this".  Having an existing task list lets you jot down the note or to-do without getting too off-track